en/docs/deploy/security/using-symmetric-encryption.md

```diff
@@ -0,0 +1,65 @@
+# Configurations Related to Symmetric Key Encryption
```

Comment:
this page was already done when doing the 5.11.0 feature docs and exists as "use-symmetric-encryption.md"